### PR TITLE
[TGA-88] fix: Add item back to IPreviewComponentProps interface

### PR DESCRIPTION
**Pull request by MarkLark86**
`item` is passed into Editor component but not Preview component, keep similar interfaces between those two allows common functionality between preview and edit.
`item` is also used by preview components in extensions from both Belga and 360info

### CODE REVIEW

**Comment by MarkLark86** [comment]
@thecalcc The `CI / test` failure is happening on `release/2.7` base branch with the same test failing:
```
HeadlessChrome 0.0.0 (Linux 0.0.0) editor3.components.embed-input should call onSubmit and reset error on success FAILED
	Expected spy unknown to have been called with [ 'resolve-value' ] but it was never called.
```

E2E is also failing in that base branch. So I don't believe I have introduced any test failures in this PR

File: scripts/apps/authoring-react/compare-articles/view-difference.tsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ interface IProps {
     profile2: IContentProfileV2;
     fieldsData1: Map<string, any>;
     fieldsData2: Map<string, any>;
+    item1: any;
+    item2: any;
```
Comment (thecalcc) [inline scripts/apps/authoring-react/compare-articles/view-difference.tsx:15]: Please type these two objects

File: scripts/apps/authoring-react/preview-article-modal.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import {Button} from 'superdesk-ui-framework/react';
 import {gettext} from 'core/utils';
 
 export function previewAuthoringEntity(
+    item: any,
```
Comment (thecalcc) [inline scripts/apps/authoring-react/preview-article-modal.tsx:9]: Please type this prop

File: scripts/core/superdesk-api.d.ts [hunk 3]
```diff
@@ -3374,6 +3374,7 @@ declare module 'superdesk-api' {
     }
 
     export interface IPreviewComponentProps<IValue, IConfig> {
+        item: any;
```
Comment (thecalcc) [inline scripts/core/superdesk-api.d.ts:3377]: Please type this

**Review by thecalcc — APPROVED** [review]

**Review by thecalcc — APPROVED** [review]

**Comment by thecalcc** [comment]
> @thecalcc The `CI / test` failure is happening on `release/2.7` base branch with the same test failing:
> 
> ```
> HeadlessChrome 0.0.0 (Linux 0.0.0) editor3.components.embed-input should call onSubmit and reset error on success FAILED
> 	Expected spy unknown to have been called with [ 'resolve-value' ] but it was never called.
> ```
> 
> E2E is also failing in that base branch. So I don't believe I have introduced any test failures in this PR

Yep, I broke them, we have a fix in progress on a different branch that's not merged yet. No worries

**Pull request merged**